library/Zend/Http/Response.php

```diff
@@ -250,6 +249,17 @@ public function setStatusCode($code)
     }
 
     /**
+     * Check is correct status code
+     *
+     * @param int $code
+     * @return bool
+     */
+    public static function isStatusCode($code)
+    {
+        return is_numeric($code) && defined('static::STATUS_CODE_' . $code);
```

Comment:
@malukenho The current behavior is such that it doesn't allow custom status codes, either.

I'm wondering if we should keep enforcing that or not.